pkgs/development/libraries/agda/fla/default.nix

```diff
@@ -0,0 +1,14 @@
+fetchFromGitHub:
+
+let
+
+  version = "0.1";
+
+  src = fetchFromGitHub {
+    repo = "functional-linear-algebra";
+    owner = "ryanorendorff";
+    rev = "v${version}";
+    sha256 = "09ri3jmgp9jjwi1mzv4c3w6rvcmyx6spa2qxpwlcn0f4bmfva6wm";
+  };
+
+in import src
```

Comment:
Alas that doesn't work in `nixpkgs` as far as I know :( I think it's not possible to import nix files from other repos. You will have to copy the agda derivation from there.

Probably that's what the build error is complaining about.

The good news is that once this is merged, you can probably import `agda.packages.functional-linear-algebra` and override its `src` to the local source.

Reply:
Shoot, I was hoping!

I copied over the default file from the repository and updated the `src` field.

Also reran the test with the new name:

```
nix-build -E "with import ./. {}; pkgs.agda.withPackages (p: with p; [ functional-linear-algebra ])"
```